crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -389,6 +389,14 @@ def prepare_nodes_for_os_upgrade
     end
 
     commit_and_check_proposal
+
+    # FIXME: upgrade of the nodes needs to be correctly orchestrated, to upgrade nodes one by one
+    # Now we're adding this only only to imitate original autoyast-based upgrad and to
+    # have current upgrade CI job working
+    upgrade_nodes.each do |node|
+      Rails.logger.info("initiating upgrade of node #{node.name}")
+      node.ssh_cmd("/usr/sbin/crowbar-upgrade-os.sh")
```

Comment:
so this script is actually doing nothing, just imitating? or do we have it in git somewhere?